frontend/src/utils.js

```diff
@@ -70,3 +70,20 @@ export function getStockLogoUrl(ticker) {
     stockLogoUrl.searchParams.append("token", import.meta.env.VITE_FINNHUB);
     return stockLogoUrl.href;
 }
+
+export function getMarketNewsUrl() {
+    const marketNewsUrl = new URL("https://finnhub.io/api/v1/news");
+    marketNewsUrl.searchParams.append("category", "forex");
+    marketNewsUrl.searchParams.append("minId", "10");
+    marketNewsUrl.searchParams.append("token", import.meta.env.VITE_FINNHUB);
+    return marketNewsUrl.href;
+}
+
+export function getStockNewsUrl(currentStock) {
+    const stockNewsUrl = new URL("https://finnhub.io/api/v1/company-news");
+    stockNewsUrl.searchParams.append("symbol", currentStock);
+    stockNewsUrl.searchParams.append("from", "2023-07-01");
```

Comment:
is there a reason this is hardcoded to July 1? Did you mean to get news within the last month? It would be better to use a variable to compute this based on current date